english/cricket/if-you-love-cricket/ifyoulovecricket.md

```diff
@@ -1 +1,3 @@
-Love cricket? [Go Here!!](../cricket.md).
\ No newline at end of file
+Love cricket? [Go Here!!](../cricket.md).
+
+There are crickets in the [Socket](../socket.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.